src/components/controls/textbox/Textbox.js

```diff
@@ -91,13 +92,17 @@ class Textbox extends React.Component {
     super(props);
 
     this.state = {
-      currentValue: ''
+      currentValue: props.value
```

Comment:
I guess what was there wouldn't work either (the empty quotes) if someone wanted to provide a default value.